lstm.py

```diff
@@ -101,18 +115,41 @@ def add_model(self, input_data):
         return sentence_logits
 
 
+    def add_perplexity_op(self, sentence_logits):
+        # TODO is this redundant with add_loss_op? Maybe be incurring
```

Comment:
Right. That's what I thought too but I didn't go through the exercise of writing out the equations. It may work out just fine but I figured to just run with this for now.